app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -62,6 +63,12 @@ interface ICloneGithubRepositoryProps {
    * available for cloning.
    */
   readonly onRefreshRepositories: (account: Account) => void
+
+  /**
+   * Called when a key event occurs on a clone-able repository
+   * list item.
```

Comment:
I think we can just copy and paste the documentation from the `FilterList` component's `onItemClick` prop here.